crates/sui-adapter/src/adapter.rs

```diff
@@ -214,13 +218,18 @@ pub fn run_metered_move_bytecode_verifier_impl(
             ]
             .contains(&e.major_status())
             {
+                metrics.verifier_timeout_metrics.with_label_values(&[BytecodeVerifierMetrics::MOVE_VERIFIER_TAG, BytecodeVerifierMetrics::TIMEOUT_TAG]).inc();
                 return Err(SuiError::ModuleVerificationFailure {
                     error: "Verification timedout".to_string(),
                 });
             };
         } else {
-            sui_verify_module_metered(protocol_config, module, &BTreeMap::new(), meter)?
+            sui_verify_module_metered(protocol_config, module, &BTreeMap::new(), meter).map_err(|err|{
+                metrics.verifier_timeout_metrics.with_label_values(&[ BytecodeVerifierMetrics::SUI_VERIFIER_TAG, BytecodeVerifierMetrics::TIMEOUT_TAG]).inc();
```

Comment:
Note that these are not all timeouts, currently.

Reply:
Yes. But for this purpose we treat the failure as a timeout pending when we disambiguate.

Reply:
Got it, is that also a "soon" follow-up? (like the timing PR?)